src/modules/user/user.repository.ts

```diff
@@ -0,0 +1,24 @@
+import { PrismaClient, User } from '@prisma/client';
+const prisma = new PrismaClient();
+export interface UserRepository {
```

Comment:
Remove, no need for definitions if there is only one way of doing concrete implementation, we don't plan on doing a v2 any time soon.